src/components/Profile/ProfileSection.tsx

```diff
@@ -12,10 +14,78 @@ export default function ProfileSection({
   isMyProfile = false,
 }: ProfileSectionProps) {
   const [isFollow, setIsFollow] = useState(false);
+  const [loading, setLoading] = useState(false);
+  const { accessToken } = useAuthStore();
+  const [followersCount, setFollowersCount] = useState(
+    user?.followers.length || 0
+  ); // 팔로워 수 상태 추가
+  const [followId, setFollowId] = useState<string | null>(null);
+
+  useEffect(() => {
+    const myId = useAuthStore.getState().user?._id;
```

Comment:
이건 useEffect가 아닌 외부로 빼셔도 될 것 같아요!
그래서 getState가 아닌 아래와 같이 수정도 가능할 것 같습니다

```suggestion
    const myId = useAuthStore((state) => state.user)?._id;
```